lib/index.js

```diff
@@ -21,10 +21,24 @@ async function getLatestVersions(name) {
   }
 }
 
+async function getLatestTag(name) {
+  const { stdout } = await execAsync(`npm view ${name} dist-tags --json`);
+  try {
+    return JSON.parse(stdout);
+  } catch (err) {
+    throw new Error(`Failed to parse output from NPM view - ${err.toString()}`);
+  }
+}
+
 async function getLatestVersion(name, wanted) {
   const versions = await getLatestVersions(name);
+  const { latest } = await getLatestTag(name);
```

Comment:
Something really small but if this named getLatestTag, it should probably already return latest destructured